docker/docs/startup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# Clone Chain Core
+git clone https://github.com/chain/chain.git $CHAIN
```

Comment:
You might consider not cloning the repo, but just copying the repo from the local fs.